troubleshooting/index.md

```diff
@@ -7,6 +7,10 @@ This document provides troubleshooting information for services and functionalit
     * [SAT/HSM/CAPMC Component Power State Mismatch](known_issues/component_power_state_mismatch.md)
     * [HMS Discovery job not creating RedfishEndpoints in Hardware State Manager](known_issues/discovery_job_not_creating_redfish_endpoints.md)
     * [initrd.img.xz not found](known_issues/initrd.img.zx_not_found.md) 
+### Grafana Dashboards
```

Comment:
The Grafana changes to this file do not seem related to this PR. They should be in with the Grafana PR, not this one.